src/client/containers/PullRequestsPaginated/index.js

```diff
@@ -35,14 +35,23 @@ class PullRequestsPaginated extends Component {
     this.props.dispatch(this.props.fetchData(args))
   }
 
+  componentWillReceiveProps(nextProps: Props) {
+    if (this.props.repo !== nextProps.repo) {
+      const args = {
+        ...this.getArguments(), repo: nextProps.repo,
+      }
+      this.props.dispatch(this.props.fetchData(args))
+    }
+  }
+
   getArguments = (): Object => ({
     pageSize: this.props.pageSize,
     limit: this.props.pageSize,
     offset: this.props.activePage > 0 ? this.props.pageSize * (this.props.activePage - 1) : 0,
     page: this.props.activePage,
     orderBy: this.props.orderBy,
     branch: this.props.branch,
-    repo: this.props.repo,
+    repo: this.props.repo || -1,
```

Comment:
It can't be `null`, as it's a variable which directly passed to graph query and `null` is incompatible with GraphQl `Int` type. Also, we need to figure out the consistency of our Graph API responses, because in some cases for nonexistent repositories it returns null, in other exception. So here requesting of the repository with `0` returns error `No identifying selector given` , but for negative or positive repositories `id`'s which are still not existent it returns `null` response without any error.

So this is workaround so far...